bundles/com.espressif.idf.launch.serial.ui/src/com/espressif/idf/launch/serial/ui/internal/CMakeMainTab2.java

```diff
@@ -550,7 +624,9 @@ public void performApply(ILaunchConfigurationWorkingCopy configuration) {
 
 			wc.setAttribute(IDFLaunchConstants.ATTR_JTAG_FLASH_ARGUMENTS, jtagArgumentsField.getText());
 			wc.setAttribute(IDFLaunchConstants.ATTR_SERIAL_FLASH_ARGUMENTS, uartAgrumentsField.getText());
+			wc.setAttribute(IDFLaunchConstants.ATTR_SERIAL_FLASH_ARGUMENTS_VIEW, uartAgrumentsField.getText());
 			wc.setAttribute(IDFLaunchConstants.ATTR_DFU_FLASH_ARGUMENTS, dfuArgumentsField.getText());
+			wc.setAttribute(IDFLaunchConstants.ATTR_DFU_FLASH_ARGUMENTS_VIEW, dfuArgumentsField.getText());
```

Comment:
I think should avoid these *VIEW fields. It complicates maintenance. I think it was a workaround in the first version of the PR that can be removed with a variables resolver